### ISSUE TITLE
Loss is always increasing in the new network

### ISSUE THREAD

**Issue by iraihankabir** [iraihankabir/DeepLearning#2]
* `Loss` is always **increasing** in the new network
* In the previous **Network** the `loss` was always decreasing.

> One thing I noticed the the accuracy is sometime high and sometime low if I run the 
> program again. I think it's because the **training set** is different for the network each time one 
> run the program.
> Our target is to reduce the loss. **Right?** . Then why the network increasing the loss?

**Comment by iraihankabir** [comment]
Although loss is increasing the accuracy good. How this is working?

**Comment by ananno** [comment]
Check the output of the network individually. Try to calculate the accuracy by hand to double check the accuracy calculation function is correct.

**Comment by iraihankabir** [comment]
I got it... 
Because of the speed I couldn't notice the decreasing first time.
